src/macros/macros.js

```diff
@@ -51,7 +51,7 @@ module.exports = class Macros {
         e.preventDefault()
         const sess = Session.list()[idx]
         if (sess && sess !== Session.focused()) {
-          Bus.emit(Bus.events.FOCUS, sess)
+          Bus.emit(Bus.FOCUS, sess)
```

Comment:
This is not correct https://github.com/elanthia-online/illthorn/blob/d47c2426cf794429140ec46ecb59b5cc363ef393/src/bus/index.js#L5

Reply:
Do you mean the change is not correct, or the event listed is not correct?

Reply:
It should be `events.FOCUS`, there is no `Bus.focus`

Reply:
I made a guess that's what you meant, so this should already be fixed.